CRM/Core/DAO/AllCoreTables.php

```diff
@@ -254,6 +254,17 @@ public static function getTableForClass($className) {
       self::tables());
   }
 
+  /**
+   * Convert the entity name into a table name.
+   *
+   * @param string $entityBriefName
+   *
+   * @return FALSE|string
+   */
+  public static function getTableForEntityName($entityBriefName) {
+    return self::getTableForClass(self::getFullName($entityBriefName));
+  }
```

Comment:
This new function seems unrelated to this PR? Was it for something else?

Reply:
Hmm - the change in hook means we don't need it for this - because now we have a class name not an  entity name